spec/jobs/alma_dump_transfer_job_spec.rb

```diff
@@ -75,7 +75,7 @@
 
         expect(dump.dump_files.count).to eq 2
         expect(dump.dump_files.map(&:dump_file_type).map(&:constant).uniq).to eq ["BIB_RECORDS"]
-        expect(dump.dump_files.first.path).to eq File.join(MARC_LIBERATION_CONFIG['data_dir'], filename1)
+        expect(dump.dump_files.last.path).to eq File.join(MARC_LIBERATION_CONFIG['data_dir'], filename1)
```

Comment:
@christinach the order of the files changed here.  I am unsure if this order is important to processing.  If it is then I can work to correct the order

Reply:
I seems that this order changes based on the machine the test is run on.  CI still sees file1 first.  I changed it to contain_exactly, but can look into the order.